docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -557,3 +557,80 @@ JSON::
 }
 ----
 ====
+== Policy & Data Source URL formats
+
+The `policy` and `data` fields in the configuration represent the URI of the policy and data sources, respectively. The following formats are supported:
+
+=== Local File
+
+A local file path scheme, such as the following, may be utilized. 
+
+* `file://<path>:`
+
+You may also use the `file::` prefix to specify a local file path.
+
+* `file::<path>:`
+
+NOTE: In both formats the path can be absolute, if a leading `/` is included, or relative without a leading `/`, to the current working directory.
+
+_Examples_:
+
+  - `file:///home/user/example/policy`
+  - `file::../example/policy`
+
+=== Git Repository
+
+A Git Hub or GitLab repository URL may be utilized:
+
+* `github.com/<org>/<repo>.git?ref=<reference>//<path>`
+* `git@github.com:<org>/<repo>.git?ref=<reference>//<path>`
+
+NOTE: `gitlab` can be used in place of `github` for GitLab repositories.
+
+You may also use the `git::` prefix to specify a Git repository URL. The following formats are supported:
+
+* `git::git@example.com:<user>/<repo>.git?ref=<reference>//<path>`:
+* `git::example.com/<org>/<repo>.git?ref=<reference>//<path>`:
```

Comment:
Again the : char, is an asciidoc formatting thing?

Reply:
Thanks for catching this. Initially theses were headings, but in editing things, they  were moved around. I'll remove the : char